### DISCUSSION TITLE
DarkReader does not apply custom CSS on initial start up.

### DISCUSSION THREAD

**Discussion by Zer0zzy** [darkreader/darkreader#12776] [Q&A]
I have made custom modifications to a lot of sites. Many of these modifications do not apply when I start up the browser. The generic darkreader template applies, but not any of the custom stuff I added. For that, I have to manually go in and select Settings > Advanced > Dev Tools > Apply. Am I missing something here? 

I did a quick look through Discussions, so it is possible I missed another post. I have a hard time believing I am the first to encounter this.

**Comment by Hets1ng** [comment]
I have the same problem as you.
https://github.com/darkreader/darkreader/discussions/7069. One contributor answered in this discussion. But it was 2 years ago. I dont know if there is a solution now.

**Comment by rahulharpal1603** [comment]
I also have the same issue, it's really frustrating.

**Comment by GhostOps77** [comment]
Is there gonna be a solution for this?

**Comment by Myshor** [comment]
As one of the contributors I will give some info about this as this subject comes back repeatedly.

Developer tools are (should) be used to try and apply some fixes only when you work on some fix. They are (should) not be used for permament fixes. That's why every browser restart the default (or newest if that option is checked) sitefixes are applied.

There are already some issues submitted for that and as I can see @alexanderby  is not willing to change the behavior for that. However as you can see I tagged him so maybe he will give some more info about this.
